agent_base:
  lr: 0.00005
  replay_memory_initial_size: -1
  replay_memory_max_size: 10000
  discount_factor: 0.99
  batch_size: 64
  n_episodes: 1000
  max_episode_steps: 500
  actor_critic_model_version: 'v1'
